pkg/generate/config/config.go

```diff
@@ -86,6 +94,11 @@ type ResourceConfig struct {
 	ListOperation *ListOperationConfig `json:"list_operation,omitempty"`
 }
 
+// resource fields that map to custom operation
+type CustomUpdateOperationFields struct {
```

Comment:
Let's call this type name `CustomUpdateOperationConfig` to match the other type names.

Reply:
Updated the type name.